src/pages/tasks/NewTaskPage.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import styles from '../../styles/styles';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as ErrorUtils from '../../libs/ErrorUtils';
+import Form from '../../components/Form';
+import TextInput from '../../components/TextInput';
+import Permissions from '../../libs/Permissions';
+import Button from '../../components/Button';
+import ROUTES from '../../ROUTES';
+
+// TO-DO: Create Avatar button for Assignee and Room Selectors
+// TO-DO: Call AssignTask with all the appropriate Data
+
+const propTypes = {
+    /** Beta features list */
+    betas: PropTypes.arrayOf(PropTypes.string),
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    betas: [],
+};
+
+// NOTE: This page is going to be updated in https://github.com/Expensify/App/issues/16855, this is just a placeholder for now
+const NewTaskPage = (props) => {
+    // const [assignee, setAssignee] = React.useState(null);
+
+    /**
+     * @param {Object} values - form input values passed by the Form component
+     * @returns {Boolean}
+     */
+    function validate(values) {
+        const errors = {};
+
+        if (!values.taskTitle) {
+            // We error if the user doesn't enter a task name
+            ErrorUtils.addErrorMessage(errors, 'taskTitle', props.translate('newTaskPage.pleaseEnterTaskName'));
+        }
+
+        if (!values.taskAssignee) {
+            // We error if the user doesn't enter a task assignee
+            ErrorUtils.addErrorMessage(errors, 'taskAssignee', props.translate('newTaskPage.pleaseEnterTaskAssignee'));
+        }
```

Comment:
Added a comment above, but they either need a task assignee or a place where the task is shared in